static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -95,16 +95,16 @@ define([
         $('.vjs-fullscreen-control', player.el()).attr('aria-label', 'video fullscreen');
     }
 
-    function createVideoPlayer(el, options) {
+    function createVideoPlayer(el, options, duration) {
```

Comment:
I think `duration` will be `undefined` here as we haven't updated how we [call](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/media/main.js#L238) `createVideoPlayer` to pass in duration...